packages/web-app-files/src/components/Shares/SharedWithMeSection.vue

```diff
@@ -25,6 +25,7 @@
       :header-position="fileListHeaderY"
       :sort-by="sortBy"
       :sort-dir="sortDir"
+      :grouping-settings="groupingSettings"
```

Comment:
Since `groupingSettings` is not a required prop, you could omit it here? Seems like you pass a basically empty object below?